pkg/docker/images.go

```diff
@@ -1,20 +1,23 @@
 package docker
 
 const (
-	ImageOperator          = "kubedb/operator"
-	ImagePostgresOperator  = "kubedb/pg-operator"
-	ImagePostgres          = "kubedb/postgres"
-	ImageMySQLOperator     = "kubedb/mysql-operator"
-	ImageMySQL             = "library/mysql"
-	ImageElasticOperator   = "kubedb/es-operator"
-	ImageElasticsearch     = "kubedb/elasticsearch"
-	ImageElasticdump       = "kubedb/elasticdump"
-	ImageMongoDBOperator   = "kubedb/mongodb-operator"
-	ImageMongoDB           = "library/mongo"
-	ImageRedisOperator     = "kubedb/redis-operator"
-	ImageRedis             = "library/redis"
-	ImageMemcachedOperator = "kubedb/mc-operator"
-	ImageMemcached         = "library/memcached"
+	ImageOperator              = "kubedb/operator"
+	ImagePostgresOperator      = "kubedb/pg-operator"
+	ImagePostgres              = "kubedb/postgres"
```

Comment:
Isn't it better this way?
```
ImageOperator              = "/operator"
ImagePostgresOperator      = "/pg-operator"
ImagePostgres              = "/postgres"
```
Then we can simply add docker-registry.

Reply:
:+1:

Reply:
What about this?
```
ImageOperator              = "operator"
ImagePostgresOperator      = "pg-operator"
ImagePostgres              = "postgres"
```

We can keep here just image name